### PR TITLE
🧹 some more `scaffolder` deprecations

### PR DESCRIPTION
**Pull request by benjdlambert**
More info in changesets

### CODE REVIEW

File: plugins/scaffolder/api-report.md [hunk 1]
```diff
@@ -238,6 +238,31 @@ export interface RepoUrlPickerUiOptions {
...
+    TemplateCardComponent?:
+      | ComponentType<{
+          template: TemplateEntityV1beta2;
+        }>
+      | undefined;
+    TaskPageComponent?: ComponentType<{}>;
+  };
+  groups?: Array<{
+    title?: string;
+    titleComponent?: React_2.ReactNode;
```
Comment (freben) [inline plugins/scaffolder/api-report.md:261]: this would probably be nice to simplify into just one prop

File: plugins/scaffolder/api-report.md [hunk 2]
```diff
@@ -458,10 +465,10 @@ export type TemplateParameterSchema = {
 // @public (undocumented)
 export const TemplateTypePicker: () => JSX.Element | null;
 
+// Warning: (ae-forgotten-export) The symbol "ScaffolderUseTemplateSecrets" needs to be exported by the entry point index.d.ts
+//
```
Comment (freben) [inline plugins/scaffolder/api-report.md:469]: should export this one too

**Review by freben — APPROVED** [review]
Just minor nits

File: plugins/scaffolder/api-report.md [hunk 3]
```diff
@@ -238,6 +238,31 @@ export interface RepoUrlPickerUiOptions {
   };
 }
 
+// Warning: (ae-missing-release-tag) "RouterProps" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export type RouterProps = {
+  TemplateCardComponent?:
+    | ComponentType<{
+        template: TemplateEntityV1beta2;
```
[the remark this answers is not in the document]
Comment (benjdlambert) [reply plugins/scaffolder/api-report.md:247]: yeah a type alias would be good, thinking that when we strip out V1beta2 in two weeks we should set that up and have a look what we can do when we get to `beta4` or  `v1` :)

File: plugins/scaffolder/api-report.md (hunk 1 above)
Comment (benjdlambert) [reply plugins/scaffolder/api-report.md:261]: Not sure what you mean here? `title?: string | React.ReactNode`?

**Pull request merged**